### PR TITLE
Team Page Revamp

### PR DESCRIPTION
**Pull request by ritam2358**
This PR has the following changes:
1. New data files were created for each subsystem for the new layout of the team page
2. Different sections were added for each subsystem.
3. Team Video responsive view bug was fixed.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -75,5 +73,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+</body>
+
+</html>
```
Comment (shubhamkorde) [inline public/index.html:78]: It is a good practice to add space at the end of file.

File: src/index.js [hunk 2]
```diff
@@ -7,4 +7,4 @@ ReactDOM.render(
     <App />
   </React.StrictMode>,
   document.getElementById("root")
-);
+);
```
Comment (shubhamkorde) [inline src/index.js:10]: Please check what files you are pushing. This is mostly an auto-lint-formatter change. Never use ```git add .```

File: src/views/TeamPage/components/Subsystems/Business.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                                activities of the team. We manage the expenditures and fundings of the team.
+                                We represent the team before our sponsors and try to procure them.
+                                We maintain the team website and pages on the different social media platforms.
+                                The main aim of the business subsystem is to ensure that the team gets sufficient
+                                fundings and has an excellent outreach to ensure the smooth functioning of the other
+                                three subsystems.
+                            </p>
+                        </Card.Text>
+                    </Card.ImgOverlay>
+                </Card> */}
```
Comment (shubhamkorde) [inline src/views/TeamPage/components/Subsystems/Business.js:32]: Delete this if not necessary

File: src/views/TeamPage/components/Subsystems/Electrical.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+} from "react-bootstrap";
+import electrical from "assets/img/electricaltransparent.jpg"
+import electricalteam from "../../../../data/ElectricalTeam.json";
+import "./Subsys.css"
+
+const Electrical = () => {
+    return (
+        <div>
+            <Container>
+                {/* <Card className="bg-dark text-white">
```
Comment (shubhamkorde) [inline src/views/TeamPage/components/Subsystems/Electrical.js:16]: Same as above

File: src/views/TeamPage/components/Subsystems/Mechanical.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import {
+    Card,
+    Container,
+    Row,
+    Col
+} from "react-bootstrap";
+import mechanical from "assets/img/mechanicaltransparent.jpg"
```
Comment (shubhamkorde) [inline src/views/TeamPage/components/Subsystems/Mechanical.js:8]: A better way to do this is to change the opacity of the image. It gives more control and is easier.

File: src/index.js (hunk 2 above)
Comment (ritam2358) [reply src/index.js:10]: noted. Will be careful from now on

File: public/index.html (hunk 1 above)
Comment (ritam2358) [reply public/index.html:78]: ok, noted

File: src/views/TeamPage/components/Subsystems/Electrical.js (hunk 4 above)
Comment (ritam2358) [reply src/views/TeamPage/components/Subsystems/Electrical.js]: fixed in the latest commit

File: src/views/TeamPage/components/Subsystems/Business.js (hunk 3 above)
Comment (ritam2358) [reply src/views/TeamPage/components/Subsystems/Business.js]: fixed in the latest commit

File: src/views/TeamPage/components/Subsystems/Subsys.css [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+  padding: 15px;
+  margin-bottom: 20px;
+  box-shadow: 0px 2px 15px rgba(53, 35, 35, 0.2);
+}
+
+.subsystem-text {
+  font-size: 18px;
+  font-family: var(--font-primary);
+  font-weight: 600;
+  /* color: var(--color-secondary); */
```
Comment (shubhamkorde) [inline src/views/TeamPage/components/Subsystems/Subsys.css:25]: The default text is too faint. Change it color-primary.

**Review by shubhamkorde — APPROVED** [review]
Impressive Work! Ready to be merged except for 2 changes. 1) Address the comment for the text color 2) The responsive width is set at large i.e. the design changes on once we make the screen size smaller than the large view-port width. Also on mobile screen the images are left aligned. Please sort this issue.

File: src/views/TeamPage/components/Subsystems/Subsys.css (hunk 6 above)
Comment (ritam2358) [reply src/views/TeamPage/components/Subsystems/Subsys.css]: done

File: src/views/TeamPage/components/Subsystems/Mechanical.js (hunk 5 above)
Comment (ritam2358) [reply src/views/TeamPage/components/Subsystems/Mechanical.js:8]: changed.

**Comment by ritam2358** [comment]
Screenshot from 2021-06-10 00-13-46

**Pull request merged**